ansible/vars/csm_packages.yml

```diff
@@ -31,6 +31,7 @@ ncn_csm_sles_packages:
   - cray-cmstools-crayctldeploy
   - csm-testing
   - curl
+  - dracut-metal-mdsquash
```

Comment:
Can we add a version constraint here without breaking the existing Ansible? Otherwise, if, for some reason, the updated `dracut-metal-musquash` wasn't available, the CFS job wouldn't fail, and the issue would sneak by unnoticed.

Reply:
Ah, right.
It turns out that we can add a version constraint without breaking it. I tested with '<', '>', '>='.